changelog.md

```diff
@@ -436,3 +463,7 @@ Republish of alpha 5 with properly building the code.
 - [jss-plugin-vendor-prefixer](https://github.com/cssinjs/jss-vendor-prefixer/blob/master/changelog.md)
 - [jss-preset-default](https://github.com/cssinjs/jss-preset-default/blob/master/changelog.md)
 - [react-jss](./docs/react-jss.md/blob/master/changelog.md)
+
+```
```

Comment:
?

Reply:
Yeah that one's confusing to me. :thinking: It definitely wasn't intentional. Glad you noticed it though.